server.py

```diff
@@ -15,29 +15,44 @@
 
 DIR = os.path.dirname(os.path.abspath(__file__))
 
+# Load the configuration.
```

Comment:
These again feel very fine-grained and fairly obvious repetition of the actual code. Can you tighten up by removing some and/or updating them with more why/context?